Shoko.Server/Databases/SQLite.cs

```diff
@@ -572,22 +573,99 @@ public void CreateDatabase()
             new DatabaseCommand(84, 1, "ALTER TABLE AnimeSeries_User ADD LastEpisodeUpdate timestamp DEFAULT NULL;"),
             new DatabaseCommand(84, 2, DatabaseFixes.FixWatchDates),
             new DatabaseCommand(85, 1, "ALTER TABLE AnimeGroup ADD MainAniDBAnimeID INT DEFAULT NULL;"),
-            new DatabaseCommand(86, 1, "ALTER TABLE AnimeEpisode_User DROP COLUMN ContractSize;"),
-            new DatabaseCommand(86, 2, "ALTER TABLE AnimeEpisode_User DROP COLUMN ContractBlob;"),
-            new DatabaseCommand(86, 3, "ALTER TABLE AnimeEpisode_User DROP COLUMN ContractVersion;"),
+            new DatabaseCommand(86, 1, DropAnimeEpisode_UserColumns),
             new DatabaseCommand(87, 1, DropAniDB_FileColumns),
             new DatabaseCommand(87, 2, AlterAniDB_GroupStatus),
-            new DatabaseCommand(87, 3, "ALTER TABLE AniDB_Character DROP COLUMN CreatorListRaw;"),
-            new DatabaseCommand(87, 4, "ALTER TABLE AniDB_Anime_Character DROP COLUMN EpisodeListRaw;"),
-            new DatabaseCommand(87, 5, "ALTER TABLE AniDB_Anime DROP COLUMN AwardList;"),
-            new DatabaseCommand(87, 6, "ALTER TABLE AniDB_File DROP COLUMN AnimeID;"),
+            new DatabaseCommand(87, 3, DropAniDB_CharacterColumns),
+            new DatabaseCommand(87, 4, DropAniDB_Anime_CharacterColumns),
+            new DatabaseCommand(87, 5, DropAniDB_AnimeColumns),
```

Comment:
Normally, renumbering these is a really bad problem. I'll let it pass because AniDB Rewrite is in such an alpha state...and it's already happened once.

Reply:
Noted